data/2017PROGRAMMING.json

```diff
@@ -0,0 +1,5416 @@
+      "Affiliations": ["TU Darmstadt"],
+      "AffiliationsString": "TU Darmstadt",
+      "Authors": ["miramezini"],
+      "Key": "modularity-2017-talks-to-be-announced-1-",
+      "PersonsString": "Mira Mezini",
+      "Title": "Modular Composition of Reactive Applications (from aspects to events and back)",
+      "Type": "Talk",
+      "URL": "http://2017.programming-conference.org/event/modularity-2017-talks-to-be-announced-1-"
+    },
+    {
+      "Abstract": "In software development, we must always be ready to respond to changes. To reflect changes flexibly, code should be organized according to system usage. Making code modularization fully reflect system usage may seem to be too ambitious. Fortunately, tests capture system usage in a well readable form of examples. However, relating a change to the corresponding place in code remains difficult because code is organized according to technical concerns, such as objects. In this paper, we propose test driven modularization, which aims at organizing code according to tests and enforces tests to be represented in code. For this, use case modularization is adapted to accommodate expressing tests in code. Test driven modularization remains compatible with use case modularization while at the same time opens new capabilities of having different views on software based on use cases and various tests simultaneously. The results of measuring effort to comprehend user actions in user interface of scenarios and complexity of use case modules show that employing user acceptance tests in code improves code comprehension and maintenance. Also, test driven modularization reduces switching between tests and code in test driven development.",
+      "Affiliations": [
+        
```

Comment:
There might have been an error in the `json` export, because the Venue should read 'Vrije Universiteit Brussel | Building D'. This is a different venue for one of the receptions. /cc @Elmervc